main.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+
+import os
+import pandas as pd
+from time import time
+import multiprocessing as mp
+from utils import *
```

Comment:
Prefer importing utils and explicitly accessing module methods, fields and classes.
This keeps clarity of the source module and reduces collisions.

```
import utils
...


utils.visium_loader(...)
```